### PR TITLE
Glimpse: Hook up delete button

### PR DESCRIPTION
**Pull request by luk1337**

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -20,4 +20,8 @@
         <item quantity="one">%d item</item>
         <item quantity="other">%d items</item>
     </plurals>
+
+    <!-- Toast messages -->
+    <string name="file_deletion_successful">File deleted!</string>
```
Comment (luk1337) [inline app/src/main/res/values/strings.xml]: Maybe we should use plurals here to be reusable?

**Review by SebaUbuntu — APPROVED** [review]

File: app/src/main/java/org/lineageos/glimpse/fragments/MediaViewerFragment.kt [hunk 2]
```diff
@@ -98,12 +100,37 @@ class MediaViewerFragment : Fragment(
...
     private val onPageChangeCallback = object : OnPageChangeCallback() {
         override fun onPageSelected(position: Int) {
             super.onPageSelected(position)
 
             this@MediaViewerFragment.position = position
 
+            if (mediaViewerAdapter.itemCount <= 0) {
+                // No medias, bail out
+                // TODO: Do better once we support showing a specific album
+                //  from intents (dialog and such)
```
Comment (luk1337) [inline app/src/main/java/org/lineageos/glimpse/fragments/MediaViewerFragment.kt]:
```
// No medias, bail out
// TODO: Do better once we support showing a specific album
// from intents (dialog and such)
```

or 

```
// No medias, bail out
// TODO: Do better once we support showing a specific album
//       from intents (dialog and such)
```

Comment (SebaUbuntu) [reply app/src/main/java/org/lineageos/glimpse/fragments/MediaViewerFragment.kt]: Is
```
// TODO: Do better once we support showing a specific album
//       from intents (dialog and such)
```
fine too?

Comment (luk1337) [reply app/src/main/java/org/lineageos/glimpse/fragments/MediaViewerFragment.kt]: Sure.

**Pull request merged**